LoopKit/QuantityFormatter.swift

```diff
@@ -217,6 +221,14 @@ public extension HKUnit {
             return (value * scaleFactor).rounded() / scaleFactor
         }
     }
+    
+    func allValues(from lowerBound: HKQuantity, through upperBound: HKQuantity) -> [Double] {
```

Comment:
The set of values between two HKQuantities is really infinite, and reading the this signature makes me wonder about that.  The method returns the set of values between the upper and lower bounds quantized by some value (here the value derived from maxFractionDigits).  Would be nice to indicate that in the message signature.

Maybe:
* `allValuesUsingMaxFractionDigits(from lowerBound: HKQuantity, through upperBound: HKQuantity)` or 
* `allValues(from lowerBound: HKQuantity, through upperBound: HKQuantity,  using fractionalDigits: Int? = nil)`

and use `maxFractionDigits` as a default if nil is passed in?

Reply:
I'll use the first.

Reply:
I would prefer the latter, to retain the ability to configure, and add documentation about the behavior when `nil` is passed in, that it uses `maxFractionDigits`